config/locales/ru.yml

```diff
@@ -85,28 +85,6 @@ ru:
       index:
         title: "Редактировать настройки"
         save: "Сохранить настройки"
-    recryptrequests:
-      index:
-        title: "Запросы на перешифрование"
-        user: "Пользователь"
-        admin_required: "Необходимы права администратора"
-        recrypt: "Перешифровать"
-      request:
-        request_sent: "Запрос на перешифрование ваших групповых паролей был отправлен. Подождите, пока администратор не перешифрует ваши пароли."
-      uncrypterror:
```

Comment:
I think this part is still required, isn't it?
so if the user has changed his ldap password, cryptopus wants to recrypt the private key with the new password.